benchmark/src/jmh/java/io/opentelemetry/benchmark/IgnoredTypesMatcherBenchmark.java

```diff
@@ -1,52 +0,0 @@
-/*
- * Copyright The OpenTelemetry Authors
- * SPDX-License-Identifier: Apache-2.0
- */
-
-package io.opentelemetry.benchmark;
-
-import io.opentelemetry.instrumentation.api.config.Config;
-import io.opentelemetry.javaagent.tooling.ignore.AdditionalLibraryIgnoredTypesConfigurer;
-import io.opentelemetry.javaagent.tooling.ignore.IgnoredTypesBuilderImpl;
-import io.opentelemetry.javaagent.tooling.ignore.IgnoredTypesMatcher;
-import java.util.concurrent.TimeUnit;
-import net.bytebuddy.description.type.TypeDescription;
-import net.bytebuddy.matcher.ElementMatcher;
-import org.openjdk.jmh.annotations.Benchmark;
-import org.openjdk.jmh.annotations.BenchmarkMode;
-import org.openjdk.jmh.annotations.Fork;
-import org.openjdk.jmh.annotations.Measurement;
-import org.openjdk.jmh.annotations.Mode;
-import org.openjdk.jmh.annotations.OutputTimeUnit;
-import org.openjdk.jmh.annotations.Warmup;
-
-@Fork(1)
-@Warmup(iterations = 3, time = 1)
-@Measurement(iterations = 10, time = 1)
-@OutputTimeUnit(TimeUnit.MICROSECONDS)
-@BenchmarkMode(Mode.AverageTime)
-public class IgnoredTypesMatcherBenchmark {
```

Comment:
this one seems worth keeping (cc @mateuszrzeszutek)

Reply:
Then I will ask some clarification about what it does :) What is the difference between `springType` and `appType`?

Reply:
The length of the matched prefix -- `com.example.myapp.Main` is dropped after `com.`, `org.springframework.test.SomeClass` after `org.springframework.`. Maybe there's a better example now, but when I wrote this benchmark and compared against the previous implementation it had to be spring - there were additional ifs, nesting and string comparisons for spring classes.

Reply:
I restored this benchmark